spring-integration-ftp/src/test/java/org/springframework/integration/ftp/config/FtpOutboundGatewayParserTests-context.xml

```diff
@@ -33,6 +33,7 @@
 							  command-options="-1 -f"
 							  expression="payload"
 							  order="1"
+							  mode="APPEND"
```

Comment:
So, we had had a `mode` attr before, but didn't parse it and there was no setter on the matter...
Merging with cherry-picking without docs.